db/tables.sql

```diff
@@ -121,4 +121,12 @@ CALL add_diner('wailunoob', 'Lim Wai Lun', 'wailunoob');
 CALL add_diner('jackbuibui', 'Jack Chen', '01234567');
 CALL add_rowner('Macdonalds', 'Ronald', 'abcdefgh');
 CALL add_rowner('KFC', 'Kentucky', 'kfckfckfc');
-CALL add_rowner('Dominoes', 'Domino', 'domiyess');
\ No newline at end of file
+CALL add_rowner('Dominoes', 'Domino', 'domiyess');
+
+-- Sample testing for Top 3 Restaurants --
+INSERT INTO Users(uname,name, pass) values  ('kh', 'kayheen', '123'), ('wy', 'wenyi', '456');
+INSERT INTO Diners values ('kh');
+INSERT INTO RestaurantOwners values ('wy');
+INSERT INTO OwnedRestaurants values ('wy', 'fudstreet', 'Popeyes'), ('wy', '1street', 'Bibimbap');
+INSERT INTO HasTimeSlots values ('19/10/2019', '10:00', 'Popeyes', 'fudstreet', 10), ('19/10/2019', '12:00', 'Popeyes', 'fudstreet', 10), ('19/10/2019', '14:00', 'Bibimbap', '1street', 10);
+INSERT INTO ReserveTimeSlots values ('19/10/2019', '10:00', 'Popeyes', 'fudstreet', 'kh', 'gr8', '3', '2', TRUE), ('19/10/2019', '12:00', 'Popeyes', 'fudstreet', 'kh', 'gr9', '4','3', TRUE), ('19/10/2019', '14:00', 'Bibimbap', '1street', 'kh', 'gr10', '5', '2', TRUE);
```

Comment:
Really small thing but it's called `ReserveTimeslots` and `HasTimeslots` hahaha (doesn't affect anything but it'll look standardised!)